### ISSUE TITLE
Github Eventizer

### ISSUE THREAD

**Issue by mschoenlaub** [chaoss/grimoirelab-cereslib#54]
Hi there,

I am in the phase of doing some research before starting my Master's Thesis.
I have done an extensive literature review already, however, more often than not, used Tools were not open sourced or lack accessibility (thinking of CHAMELEON for clustering, or Understand for static analysis).

I am now planning to use parts of grimoirelab to extract a collaboration graph.

Anyways, I do think that I am going to write a Github eventizer, to extract the necessary information from PRs extracted from the repository. Would there be any interest in having this emrged upstream? If so, I'd fork the project rather than adding it to my script. 

BR,

Manuel

**Comment by sduenas** [comment]
Hi @mschoenlaub, feel free to contribute with a GitHub eventizer. That'll be great. Just please remember that you should include some unit test to prove your code works before we merge it on the main branch.

Thanks!

**Comment by sduenas** [comment]
Closing this ticket due to inactivity. Feel free to open a PR contribute with this.
